### PR TITLE
Fix color of cluster labels in Mermaid flowcharts

### PR DESCRIPTION
**Pull request by pawamoy**
See https://github.com/squidfunk/mkdocs-material/discussions/3618

Without this change, a rule like `color: #333` would be applied on clusters labels (`.cluster span`), making these difficult to read with the dark theme enabled.

I have other files that were modified, I think I should commit them as well (at least the ones already tracked)? Let me know.

```
 M material/base.html
 M material/overrides/main.html
 M package-lock.json
?? material/assets/javascripts/bundle.js
?? material/assets/javascripts/bundle.js.map
?? material/assets/javascripts/workers/search.js
?? material/assets/javascripts/workers/search.js.map
?? material/assets/stylesheets/main.css
?? material/assets/stylesheets/main.css.map
?? material/assets/stylesheets/palette.css
?? material/assets/stylesheets/palette.css.map
?? material/overrides/assets/javascripts/bundle.js
?? material/overrides/assets/javascripts/bundle.js.map
?? material/overrides/assets/stylesheets/main.css
?? material/overrides/assets/stylesheets/main.css.map
```

### CODE REVIEW

**Comment by squidfunk** [comment]
Thanks for the PR. Please go through the [theme build process](https://squidfunk.github.io/mkdocs-material/customization/#building-the-theme) – it will remove all non-production files (which you currently see in your working copy). Please don't include the `package-lock.json`.

**Comment by squidfunk** [comment]
Could you please also share the Markdown that this PR fixes?

**Comment by pawamoy** [comment]
Example flowchart:

````md
```mermaid
flowchart LR

client["client"]
load_balancer["load balancer"]
api1["API (replica 1)"]
api2["API (replica 2)"]
apin["API (replica N)"]

client --- load_balancer
subgraph openshift
    subgraph service
        api1 --- api2 --- apin
    end
    load_balancer --- service
end
```
````

```mermaid
flowchart LR

client["client"]
load_balancer["load balancer"]
api1["API (replica 1)"]
api2["API (replica 2)"]
apin["API (replica N)"]

client --- load_balancer
subgraph openshift
    subgraph service
        api1 --- api2 --- apin
    end
    load_balancer --- service
end
```

Before:

chart_before

After:

chart_after

**Comment by pawamoy** [comment]
I also have:

```
## fix-mermaid-subgraph-labels-color...origin/fix-mermaid-subgraph-labels-color
 M material/.icons/material/LICENSE
 D material/overrides/assets/javascripts/iconsearch_index.json
```

LICENSE seems to be CR/LF related. Not sure about the iconsearch_index.json deletion.

File: src/assets/javascripts/components/content/code/mermaid/index.css [hunk 1]
```diff
@@ -92,6 +92,11 @@ marker {
   stroke: var(--md-default-fg-color--lighter);
 }
```
Comment (squidfunk) [inline src/assets/javascripts/components/content/code/mermaid/index.css:94]: Looking good! Only thing missing is a comment on what this is exactly styling, see the other comments 😉

Comment (squidfunk) [reply src/assets/javascripts/components/content/code/mermaid/index.css:94]: Explanation: the CSS is very complex, so I try to add short and concise comments that a selector is targeting, so things are not accidentally removed or forgotten to be adapted.

Comment (pawamoy) [reply src/assets/javascripts/components/content/code/mermaid/index.css:94]: Of course, I thought the comment above would suffice, but we can be even more explicit :+1:

**Comment by squidfunk** [comment]
Thanks! LGTM!

**Pull request merged**